db/compaction/subcompaction_state.h

```diff
@@ -195,16 +201,21 @@ class SubcompactionState {
   // Close all compaction output files, both output_to_penultimate_level outputs
   // and normal outputs.
   Status CloseCompactionFiles(const Status& curr_status,
+                              bool supports_per_key_placement,
```

Comment:
nit: this doesn't have to be a parameter? Can be computed inside CloseCompactionFiles().

Reply:
Oh jeez, I didn't see that someone mixed public fields at the top, along with the private fields at the bottom :(